assets/stylesheets/_vendor.scss

```diff
@@ -5,15 +5,15 @@
 .gridicon {
 	fill: currentColor;
 
-	&.needs-offset g {
+	&.needs-offset :first-child {
```

Comment:
When the whole SVG is inline rather than referenced, doesn't this also offset all nested elements that happen to be the first child of their parent?
```
&.needs-offset > g:first-child, &.needs-offset > use:first-child {
  transform: ...
}
```
should be more reliable. What do you think?

Reply:
Good catch, once again! But if the concern is to restrict to immediate descendants (which is a valid concern), then the following should suffice:

```css
&.needs-offset > :first-child {
  transform: ...
}
```

Would you prefer for me to still explicitly mention `g` and `use`?

Reply:
> Would you prefer for me to still explicitly mention g and use?

Not really needed, just more safe and explicit IMO 🙂

Reply:
No worries, I'l switch to that, then :)